src/mbyte.c

```diff
@@ -140,6 +140,7 @@ static int dbcs_head_off(char_u *base, char_u *p);
 #ifdef FEAT_EVAL
 static int cw_value(int c);
 #endif
+int get_cellwidth(int c);
```

Comment:
This is not needed because it's not a static function

Reply:
Thank you.
Fixed in commit c69bff00c240028442255f77ee861421e5100db5.